src/natcap/invest/coastal_vulnerability.py

```diff
@@ -724,25 +724,21 @@ def interpolate_shore_points(
         shapely_line_list = pickle.load(lines_pickle_file)
     lines_in_aoi_list = []
     for shapely_geom in shapely_line_list:
-        # Already clipped landlines to AOI, with max-fetch extent
-        # still want to clip lines by actual AOI, for shore point creation
+        # Already clipped landlines to AOI plus max-fetch extent
+        # still want to clip lines by actual AOI for shore point creation
         if aoi_shapely_prepped.intersects(shapely_geom):
             intersected_shapely_geom = aoi_shapely.intersection(shapely_geom)
             if not intersected_shapely_geom.is_empty:

```

Comment:
Could this be empty since there is an intersection? Shapely documentation indicates that a Point( ) would return True, is that possible here?

Reply:
@dcdenu4, the intersection could be empty: `shapely_geom` could be outside of `aoi_shapely`. That's not so uncommon because the landmass lines at this point (`shapely_geom`) were clipped to a bounding box that is aoi extent + max-fetch-distance. 

The intersection could also be a point in the very rare case that `shapely_geom` is a tangent line to `aoi_shapely`. And that's not empty, but is handled below.